src/extension/module.js

```diff
@@ -1432,14 +1432,52 @@
     }
   }
 
+  /**
+   * Registers a plugin for re-evaluation if it should be shown or hidden,
+   * and if its button should be enabled or disabled.
+   * @private
+   * @param {Sidekick} sk The sidekick
+   * @param {Plugin} plugin The plugin configuration
+   * @param {HTMLElement} $plugin The plugin
+   * @returns {HTMLElement} The plugin or {@code null}
+   */
+  function registerPlugin(sk, plugin, $plugin) {
+    // re-evaluate plugin when status fetched
+    sk.addEventListener('statusfetched', () => {
+      if (typeof plugin.condition === 'function') {
+        if ($plugin && !plugin.condition(sk)) {
+          // plugin exists but condition now false
+          sk.remove(plugin.id);
+        } else if (!$plugin && plugin.condition(sk)) {
+          // plugin doesn't exist but condition now true
+          sk.add(plugin);
+        }
+      }
+      const isEnabled = plugin.button && plugin.button.isEnabled;
+      if (typeof isEnabled === 'function' || typeof isEnabled === 'boolean') {
```

Comment:
why only `function` and `boolean`. and not for example `number` or `string` ?

Reply:
It can only be a `function`: https://github.com/adobe/helix-sidekick-extension/blob/mv3/src/extension/module.js#L35